app/models/user.js

```diff
@@ -21,5 +24,20 @@ export default Addressable.extend({
 
   fullName: Ember.computed('firstName', 'lastName', function(){
     return (this.get('firstName') + " " + this.get('lastName'));
-  })
+  }),
+
+  isInfoComplete: Ember.computed('firstName', 'lastName', 'email', 'mobile', 'title', function(){
+    return this.hasValue('firstName', 'lastName', 'email', 'mobile', 'title');
+  }),
+
+  hasValue(...args) {
+    let hasValue = true;
+    for (let i = 0; i < args.length; i ++) {
+      if(!this.get(args[i]) || !this.get(args[i]).length) {
```

Comment:
What will happen to `this.get(args[i])` if consider first_name value will be blank string ? 🤔

Reply:
This code look reusable and not related to specific model. 

What if we move this code to utility_methods file ?

CC: @Darshan-Chauhan

Reply:
@Darshan-Chauhan , Its use at just one place.. Right?